### DISCUSSION TITLE
What protocol does the driver use?

### DISCUSSION THREAD

**Discussion by andysas** [kubernetes-csi/csi-driver-smb#837] [Q&A] [answered]
What version of the SMB protocol does the driver use? I am asking since I was not able to find this information in the documentation.

**Comment by andyzhangx** [comment, marked as answer]
that depends on the smb driver kernel on the node, not related to this csi driver

**Reply by andysas** [reply]
@andyzhangx Thanks for the revert. What protocols does the SMB driver kernel on the node support? Again, just trying to get an understanding of the protocols we support while mounting windows drives on K8s cluster.

**Reply by andyzhangx** [reply]
usually the supported smb version could be 2.2, 3.0, 3.1, etc.
